torch/testing/_internal/common_nn.py

```diff
@@ -4816,6 +4883,8 @@ def __init__(self, *args, **kwargs):
         self.check_half = kwargs.get('check_half', True)
         self.check_bfloat16 = kwargs.get('check_bfloat16', False)
         self.convert_target = kwargs.get('convert_target', True)
+        self.with_tf32 = kwargs.get('with_tf32', True)
```

Comment:
Why is with_tf32 defaulted to True here, but False in NewModelTest? 

Should NewCriterionTests be affected at all by this flag?

Reply:
You are right that it is not affected. This flag is removed now.